src/main/webapp/groups.js

```diff
@@ -84,4 +84,20 @@ function createGroupResult(result) {
     return container;
 }
 
+/** When a user selects "other" topic, it displays the text input so that the user can type their topic. If the user 
+*   selects a different topic, it hides the text input.
+*/
+function checkOtherSelected(value) {
+    var otherTextBox = document.getElementById('other-topic');
+    if(value === 'other') {
+        otherTextBox.style.display = 'block';
+        otherTextBox.setAttribute("required", "");
+        otherTextBox.setAttribute("aria-required", true);
```

Comment:
What's the difference between required and aria-required?

Reply:
I read that aria-required increases accessibility and is compatible with more agents